### PR TITLE
Investigation Log and to_chat Fixes

### PR DESCRIPTION
**Pull request by Krausus**
- Fixes investigation logs using a fixed list of subjects, which included many *not* in use, and didn't include all the ones that *are* in use.
  - The subject list is now populated as info on different subjects is logged.
- Fixes a runtime that occurs whenever logging anything to the investigate logs.
  - This runtimes the moment you connect to the server. It is literally impossible to miss. Why, @tigercat2000? Why?
- Fixes a runtime that occurs when the round-start welcome message plays.
  - WHY?
- Fixes a bunch of other places where `to_chat` is called with sounds, many of which would runtime.
- Fixes `to_chat` referring to itself as `Boutput` when it intentionally crashes.

### CODE REVIEW

File: code/modules/admin/admin_investigate.dm [hunk 1]
```diff
@@ -49,50 +49,9 @@
...
-			src << browse(F,"window=investigate[subject];size=800x300")
-
-		if("atmos")
-			var/F = investigate_subject2file(subject)
-			if(!F)
-				to_chat(src, "<font color='red'>Error: admin_investigate: [INVESTIGATE_DIR][subject] is an invalid path or cannot be accessed.</font>")
-				return
-			src << browse(F,"window=investigate[subject];size=800x300")
-
-		if("ntsl")
```
Comment (TheDZD) [inline code/modules/admin/admin_investigate.dm]: Why was ntsl in here? I could've sworn I nuked this.

Comment (Krausus) [inline code/modules/admin/admin_investigate.dm]: To be clear, its presence here doesn't necessarily mean anything was being logged with the "ntsl" subject, just that it was a subject at one point in time. Maybe.

**Comment by Krausus** [comment]
There's, uh... there's a **lot** of `to_chat` calls that are going to crash now. I'm gonna look into fixing as many as I can find...

**Comment by Krausus** [comment]
That's all of the ones that stood out to me. I'd imagine there's more spots that are going to crash, but they should show up once the live server gets patched. This should be good to go.

**Pull request merged**